pinot-query-planner/src/main/java/org/apache/pinot/query/type/TypeSystem.java

```diff
@@ -30,7 +31,8 @@
  */
 public class TypeSystem extends RelDataTypeSystemImpl {
   private static final int MAX_DECIMAL_SCALE = 1000;
-  private static final int MAX_DECIMAL_PRECISION = 1000;
+  @VisibleForTesting
+  static final int MAX_DECIMAL_PRECISION = 1000;
```

Comment:
what's the reason behind this?

Reply:
I think I needed in an initial version of TypeFactoryTest.java, but I end up not needing it. Removing the change.